src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/CopyResultsRequest.cs

```diff
@@ -38,7 +38,6 @@ public class CopyResultsRequestParams : SubsetParams
     /// </summary>
     public class CopyResultsRequestResult
     {
-        public string Results { get; set; }
```

Comment:
so if the copy operation is being done in the UI process (no Linux or remote extension host), how does the UI get the string to put on the clipboard now?

Reply:
Copying from the UI is done when this setting is unchecked in Azure Data Studio's settings
![image](https://github.com/user-attachments/assets/57c27940-783c-47df-964a-044d27be3858)

When the "Prefer Providers Copy Handler" setting is unchecked, the STS endpoint that copies results will not be invoked and a different code flow will handle copying results from the UI. The logic for that is found here in the Azure Data Studio repo: [Code in screenshot](https://github.com/microsoft/azuredatastudio/blob/570c27b39edb2e536a89e7f7df20f886fc219296/src/sql/workbench/services/query/common/queryRunner.ts#L608)

![image](https://github.com/user-attachments/assets/c6e2e3bf-2d92-4d9d-b0f8-c87e7db77309)

`handleCopyRequestByProvider` is the method that allows STS to copy results, and `copySelectionToClipboard` copies query results to the clipboard from the UI.